.github/workflows/main.yml

```diff
@@ -7,8 +7,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        # I don't really understand the build matrix here...
-        build: [stable, beta, nightly, macos, windows, mingw]
+        build: [stable, beta, nightly, minimal, macos, windows, mingw]
```

Comment:
CI now fails [on `main`](https://github.com/rust-lang/flate2-rs/actions/runs/10439240311), and I have a feeling that this entry might be the cause of it?

It's surprising that it didn't fail CI before.

In any case, your help for a fix will be welcome.